easyDiffractionApp/Logic/PyQmlProxy.py

```diff
@@ -820,14 +872,53 @@ def _onExperimentDataChanged(self):
     @Slot(str)
     def addExperimentDataFromXye(self, file_url):
         print(f"+ addExperimentDataFromXye: {file_url}")
-        self._experiment_data = self._loadExperimentData(file_url)
-        self.experimentDataAdded.emit()
+
+        def outer1(obj):
```

Comment:
this and `RemoveExperiment` content can be refactored out, since these are the same (bar minor things).
I've already done that in the refactoring branch, though, so no action needed

Reply:
no longer relevant